cli/commands/logs/index.ts

```diff
@@ -0,0 +1,76 @@
+import { AwilixContainer } from 'awilix';
+import { CommandHandler } from "../..";
+import { assertExists, assertIsISOString, isValidTimeRange } from "../../utils";
+import { GetAgentLogs } from '../../utils/get.agent.logs';
+
+const SECONDS_IN_A_DAY = 86400;
+
+export default function provideLogs(
+  container: AwilixContainer
+): CommandHandler {
+  assertExists(container, 'container')
+
+  return async function logs(cliArgs: any) {
+    const getAgentLogs = container.resolve<GetAgentLogs>('getAgentLogs')
+    
+    const agentId = cliArgs.agentId
+    if(!agentId) throw new Error(`No agentId provided`)
+
+    
+    let latestTimestamp = cliArgs.before
+    let earliestTimestamp = cliArgs.after
+
+    // If no time range entered
+    if(!latestTimestamp && !earliestTimestamp) {
+      // Default to the last 24 hours
+      earliestTimestamp = new Date(Date.now() - (SECONDS_IN_A_DAY * 1000)).toISOString()
+      latestTimestamp = new Date(Date.now()).toISOString()
+    }
```

Comment:
❓ Right now the `after` and `before` params are optional with a default time range of the last 24 hours of logs. Does that make sense for this cli command?

Reply:
The logs api makes a lot of extreme trade-offs for safety and scale, and one of those trade-offs is the inability to search across minutes.  If we do this, can we make the maximum range 1 hour?  (that'd be 60 requests)